src/devices/device_models/AC_branches.jl

```diff
@@ -117,7 +134,7 @@ function branch_rate_constraint(canonical_model::CanonicalModel,
                                 device_formulation::Type{D},
                                 system_formulation::Type{S}) where {B<:PSY.ACBranch,
                                                                     D<:AbstractBranchFormulation,
-                                                                    S<:PM.AbstractActivePowerFormulation}
+                                                                    S<:PM.AbstractDCPLLModel}
```

Comment:
Should this be `AbstractActivePowerModel`

Reply:
yes, thanks for that.